plugins/LilyPad/Config.cpp

```diff
@@ -48,7 +48,7 @@ void PCSX2_overrideConfig(GeneralConfig &config_in_out)
     config_in_out.disableScreenSaver = 0;   // Not required - handled internally by PCSX2
     config_in_out.escapeFullscreenHack = 0; // Not required - handled internally by PCSX2
     config_in_out.saveStateTitle = 0;       // Not required - handled internally by PCSX2
-    config_in_out.closeHacks = 0;           // Cannot function when used by PCSX2
+    config_in_out.closeHack = 0;            // Cannot function when used by PCSX2
```

Comment:
It can be useful if you want a quick exit out of an emulator. Yes, it does say it might not do a proper cleanup, but that's been the case ever since it was introduced. I haven't found anyone ever complaining about it, but that might also be because hardly anyone has ever used it. :p